### DISCUSSION TITLE
Question about perturbation applcaition?

### DISCUSSION THREAD

**Discussion by ShahrzadH** [bowang-lab/scGPT#121] [Q&A]
Hi, Thanks very much for your great work and you effort in responding to issues veery quickly!  I have a few questions:
1- There are multiple pre-trained models, I know you recommend mostly scGPT_human for most of the downstream applications. Could you please let me know which one is more suitable for perturbations? What are differences between all pre-trained sc_GPTs (human, brain, heart, etc)?
2- What is relation between sc_GPT and GEARS? Is sc_GPT only using GEARS datasets for fine-tuning or there is more to it? I mean whether sc_GPT uses GEARS predictions as part of its training? 
Thanks for taking time to answer my questions in advance!

**Comment by subercui** [comment]
Hi, thank you for the question. 1. Basically, we currently recommend using the human model as the default go-to option for all applications including perturbation response prediction. The other organ or condition-specific models are trained using cells from the organ or of the condition. Usually, when you have a sample of that kind, say brain tissue sample, you may also have a try with the pretrained brain model. 2. We used the GEARS to load, split datasets in training, validation, and test sets, and to compute some evaluation metrics. The core training and prediction computation is solely based on our own method.

**Reply by ShahrzadH** [reply]
Thanks for your reply! Could you please advise how I can infer how CRISPR knock out of a gene will perturb the rest of the genes?

**Reply by subercui** [reply]
Hi, Could you elaborate further on your question? For example, people would use Perturb-seq or other techniques to verify the responses of perturbations, and we also show in our tutorials to train a model and to predict unseen perturbations in silico.
